diff --git a/node_modules/@keystatic/astro/dist/keystatic-astro-ui.js b/node_modules/@keystatic/astro/dist/keystatic-astro-ui.js
index 922d592..a5b3c57 100644
--- a/node_modules/@keystatic/astro/dist/keystatic-astro-ui.js
+++ b/node_modules/@keystatic/astro/dist/keystatic-astro-ui.js
@@ -1,6 +1,6 @@
 import 'react';
 import { Keystatic } from '@keystatic/core/ui';
-import { jsx } from 'react/jsx-runtime';
+import { jsx, jsxs } from 'react/jsx-runtime';
 
 const appSlug = {
   envName: 'PUBLIC_KEYSTATIC_GITHUB_APP_SLUG',
@@ -8,9 +8,13 @@ const appSlug = {
 };
 function makePage(config) {
   return function Keystatic$1() {
-    return /*#__PURE__*/jsx(Keystatic, {
-      config: config,
-      appSlug: appSlug
+    return /*#__PURE__*/jsxs("div", {
+      children: [/*#__PURE__*/jsx("style", {
+        children: "button[aria-label=\"Image\"]{display:none!important;}div[role=\"dialog\"]>div:has(>div>button+button):has(>[role=\"separator\"]),div[role=\"dialog\"]>div:has(>div>button:only-child):has(>div>button:not([aria-label])){display:none!important;}"
+      }), /*#__PURE__*/jsx(Keystatic, {
+        config: config,
+        appSlug: appSlug
+      })]
     });
   };
 }
